pkg/project/version_bundle_azure.go

```diff
@@ -8,10 +8,18 @@ var versionBundleAzure = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
 			Component:   Name(),
-			Description: "Use Release.Revision in annotation for Helm 3 compatibility.",
+			Description: "Fix bug in user values migration logic for apps.",
+			Kind:        versionbundle.KindFixed,
+			URLs: []string{
+				"https://github.com/giantswarm/cluster-operator/pull/1030",
+			},
+		},
+		{
+			Component:   Name(),
+			Description: "Changes to support making nginx-ingress-controller an optional app.",
```

Comment:
@whites11 Release PR is prepared. Does this look OK for the changelog?

Let me know when you're ready for this to be merged.

Reply:
Changelog OK for me